src/main/scala/com/gu/scanamo/ops/ScanamoInterpreters.scala

```diff
@@ -0,0 +1,130 @@
+package com.gu.scanamo.ops
+
+import cats._
+import cats.data.Xor
+import com.amazonaws.AmazonWebServiceRequest
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.services.dynamodbv2.{AmazonDynamoDB, AmazonDynamoDBAsync}
+import com.amazonaws.services.dynamodbv2.model.{UpdateItemResult, _}
+import com.gu.scanamo.request.{ScanamoDeleteRequest, ScanamoPutRequest, ScanamoUpdateRequest}
+
+import scala.concurrent.{ExecutionContext, Future, Promise}
+import scala.util.{Failure, Success}
+
+object ScanamoInterpreters {
```

Comment:
Why not just `Interpreters`? The package tells us that it is a Scanamo thing. (I need an emoji for "this is a minor point")